src/azure-cli/azure/cli/command_modules/appconfig/custom.py

```diff
@@ -2,13 +2,18 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
-from azure.mgmt.appconfiguration.models import ConfigurationStoreUpdateParameters
+from azure.mgmt.appconfiguration.models import (ConfigurationStoreUpdateParameters,
+                                                ConfigurationStore,
+                                                Sku)
 
 from ._utils import resolve_resource_group, user_confirmation
 
 
 def create_configstore(client, resource_group_name, name, location):
-    return client.create(resource_group_name, name, location.lower())
+    configstore_params = ConfigurationStore(location=location.lower(),
+                                            identity=None,
+                                            sku=Sku(name="Free"))   # hard code the sku as it is not public facing yet
```

Comment:
In the future, will sku be a parameter which can be passed by users? If yes, can we consider make `sku` in argument list for `creat_configstore()`? To make it not exposed to users now, you can use `c.ignore('sku')` to ignore sku parameter and one day sku is supported, you just need to change it to `c.argument('sku')`.

Reply:
Done

---
In reply to: [347081465](https://github.com/Azure/azure-cli/pull/11036#discussion_r347081465) [](ancestors = 347081465)

Reply:
But I haven't seen sku in your parameter list here.

Reply:
You may look at the wrong iteration. This is the latest one. [Commit 16](https://github.com/Azure/azure-cli/pull/11036/commits/9d16e2d0cdf764ea96df79fb28456a95dfeb5b4d)